apps/styleguide/src/layout/MolgenisMenu.vue

```diff
@@ -71,6 +71,25 @@ export default {
     /** session information, so we can check role permissions */
     session: Object,
   },
+  computed: {
+    permittedItems() {
+      return this.items.filter(this.permitted);
+    },
+    homeUrl() {
+      //will be first non-submenu item that is permitted
+      if (this.permittedItems.length > 0) {
+        let firstItem = this.permittedItems[0];
+        // in case it is a item with submenu and without href, find first submenu item
+        while (firstItem && firstItem.href == null && firstItem.submenu) {
+          let sub = firstItem.submenu.filter((i) => this.permitted(i));
+          firstItem = sub[0];
+        }
+        if (firstItem && firstItem.href) return firstItem.href;
+      }
+      //defaut: go home
+      return "../";
```

Comment:
why is the home '../'  ?

Reply:
this automatically would lead you to default of the schema. Agree?

Reply:
yes , by why the '../' ( instead of '/' or '', or ../../, ...)